CruCentralCoast/Resources/ResourcesVC.swift

```diff
@@ -72,38 +64,45 @@ class ResourcesVC: UIViewController {
 
 extension ResourcesVC: SegmentedControlInNavBarViewProtocol {
     func segmentedControlValueChanged(_ index: Int) {
-        self.segmentedControlIndex = index
-        switch index {
+        self.collectionView.scrollToItem(at: IndexPath(item: index, section: 0), at: .left, animated: true)
+    }
+    
+}
+
+extension ResourcesVC: UICollectionViewDataSource {
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 3
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueCell(ResourcesTableViewCollectionViewCell.self, indexPath: indexPath)
+        switch indexPath.item {
         case 0:
-            self.tableView.dataSource = self.articlesResources
+            cell.type = .articles
         case 1:
-            self.tableView.dataSource = self.videosResources
+            cell.type = .videos
         case 2:
-            self.tableView.dataSource = self.audioResources
+            cell.type = .audio
         default:
-            return
+            break
         }
-        self.tableView.reloadData()
-        self.tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: false)
+        
+        return cell
     }
     
+    func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
+        let index = Int(scrollView.contentOffset.x/self.collectionView.frame.width)
+        self.segmentedControlUpdateOnScrollDelegate?.setSegmentedControlValue(index)
+    }
 }
 
-extension ResourcesVC: UITableViewDelegate {
-        func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-            switch self.segmentedControlIndex {
-            case 0:
-                return 60
-            case 1:
-                return 60
-            case 2:
-                return 80
-            default:
-                return 44
-            }
-        }
-    
-        func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-            return self.segmentedControlInNavBarView
-        }
+extension ResourcesVC: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return collectionView.frame.size
+    }
+}
+
+protocol UpdateSegmentedControlDelegate {
```

Comment:
This naming sounds weird. Maybe CruSegmentedControlDelegate would be better. However, you are removing this, so all is good.